Skill to find dart or flutter revision from a hash (#184589)
flutter/flutter: .agents/skills/find-release/SKILL.md

---
name: find-release
description: A skill to find the lowest Dart and Flutter release containing a given commit. Use this skill whenever users ask about when a commit landed in Flutter or Dart releases, inquire about release versions for specific SHAs, or want to know if a commit is included in stable, beta, or dev channels for Flutter/Dart projects.
---

## Instructions

1.  **Extract Information:**
    - Identify the commit SHA from the user's request (e.g., `02abc57`).

>     - Identify the target channel (one of: `stable`, `beta`, `dev`). If not specified, default to `stable`.

Reviewer: I would consider telling it to just search all channels by default, or maybe search in order stable, beta, then dev until it finds one

Author: ```suggestion
    - Identify the target channel (one of: `stable`, `beta`, `dev`). If not specified, try each.
```
